lavamoat/build-system/policy.json

```diff
@@ -1913,7 +1913,6 @@
       },
       "packages": {
         "chokidar>braces": true,
-        "chokidar>fsevents": true,
```

Comment:
Hey I'm not very familiar with this file, but hope these 2 are safe to move, just flag it out to double check with you ~

Reply:
We get some discrepancies with these files based on OS and this is one that we definitely get discrepancies with.

Reply:
Gotcha !